stacker/stack.py

```diff
@@ -111,11 +111,22 @@ def requires(self):
                 requires.add(stack_fqn)
 
         # Add any dependencies based on output lookups
-        for lookup in self.lookups:
-            if lookup.type == OUTPUT_LOOKUP_TYPE_NAME:
-                d = deconstruct(lookup.input)
-                stack_fqn = self.context.get_fqn(d.stack_name)
-                requires.add(stack_fqn)
+        for variable in self.variables:
```

Comment:
in stacker v1.0 this is the only code within this function, so i don't think it needs to be broken out more